src/RobotModelUpdate.cpp

```diff
@@ -36,20 +47,123 @@ void RobotModelUpdate::reset(mc_control::MCGlobalController & controller)
   mc_rtc::log::info("Robot Update is:\n{}", robotUpdate.dump(true, true));
 
   gui.removeElements(this);
-  gui.addElement(this,
-      {"RobotModelUpdate", robot_},
-      mc_rtc::gui::Button("Reset to default",
-        [this, &ctl]()
-        {
-          resetToDefault(ctl.robot(robot_));
-        })
-  );
-
-  robotUpdate.addToGUI(gui, {"RobotModelUpdate", robot_}, "Update Robot Model", 
-       [this, &ctl]() {
-        mc_rtc::log::info("Updated robot schema");
-        updateRobotModel(ctl);
-       });
+  gui.addElement(this, {"Plugin", "RobotModelUpdate", robot_},
+                 mc_rtc::gui::Button("Reset to default",
+                                     [this, &ctl]()
+                                     {
+                                       resetToDefault(ctl.robot(robot_));
+                                       resetToDefault(ctl.outputRobot(robot_));
+                                     }),
+                 mc_rtc::gui::Button("Load Xsens config", [this, &ctl]() { configFromXsens(ctl); }));
+
+  robotUpdate.addToGUI(gui, {"Plugin", "RobotModelUpdate", robot_}, "Update Robot Model",
+                       [this, &ctl]()
+                       {
+                         mc_rtc::log::info("Updated robot schema");
+                         updateRobotModel(ctl);
+                       });
+}
+
+void RobotModelUpdate::configFromXsens(mc_control::MCController & ctl)
```

Comment:
This could be made fully configurable from `YAML`, maybe something like this:

```yaml
UpdateConfig:
  LArm_0:
    - segment1: Pelvis
      segment2: Left Upper Arm
      offset:
         translation: [0, 0, 0]
...
```

The code can then be simplified to a loop on all joints we want to modify applying the same operation each time ;)